CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [0.5.3] - 2020-07-13
```

Comment:
Why we don't have any changelogs here? Were are the changes irrelevant for the user?